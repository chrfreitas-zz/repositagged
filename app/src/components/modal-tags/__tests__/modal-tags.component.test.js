import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import ModalTags from '../index';

describe('ModalTags component', () => {
  it('should match with snapshot', () => {
    const repository = {
      id: 1,
      name: 'mongodb',
    };

    const renderer = new ShallowRenderer();
    const result = renderer.render(
      <ModalTags
        isOpen
        save={() => {}}
        close={() => {}}
        repository={repository}
      >
        Lorem ipsum dolor sit, amet consectetur adipisicing elit. Sequi, veniam.
      </ModalTags>,
    );
    expect(result).toMatchSnapshot();
  });
});
