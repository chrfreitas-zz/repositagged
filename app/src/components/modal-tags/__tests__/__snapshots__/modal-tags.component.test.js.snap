// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalTags component should match with snapshot 1`] = `
<ModalUI
  cancel={[Function]}
  confirm={[Function]}
  isOpen={true}
  title="edit tags for mongodb"
>
  <Input
    autoFocus={true}
    className="input input--full"
    onChange={[Function]}
    placeholder="Create new tags"
  />
</ModalUI>
`;
